src/df.c

```diff
@@ -58,8 +71,8 @@ static ignorelist_t *il_fstype;
 static ignorelist_t *il_errors;
 
 static bool report_inodes;
-static bool values_absolute = true;
-static bool values_percentage;
+static bool report_usage = true;
+static bool report_utilization;
```

Comment:
If config option names are changed, maybe these too?
```suggestion
static bool report_bytes = true;
static bool report_ratio;
```